app/src/test/java/org/mozilla/fenix/components/BackgroundServicesTest.kt

```diff
@@ -50,10 +50,10 @@ class BackgroundServicesTest {
         val context = mockk<Context>(relaxed = true)
 
         every { context.isInExperiment(eq(Experiments.asFeatureWebChannelsDisabled)) } returns false
-        assertEquals("urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel", BackgroundServices.redirectUrl(context))
+        // assertEquals("urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel", BackgroundServices.redirectUrl(context))
```

Comment:
This seems wrong. Why did you need to disable these tests?

Reply:
Ah this was likely because `redirectUrl` is no longer a function on `BackgroundServices`. Didn't notice this was commented out. So, we can change that to `FxaServer.redirectUrl(context)` now and it should work again.

Reply:
Oh, sorry about that I commented it so that building on TC did not complain and forgot to fix. Fixing...